src/k-provider/ovp/provider.js

```diff
@@ -98,6 +101,9 @@ export default class OVPProvider extends BaseProvider<OVPProviderMediaInfoObject
         const sessionLoader = data.get(OVPSessionLoader.id);
         if (sessionLoader && sessionLoader.response) {
           mediaConfig.session.ks = sessionLoader.response;
+          if (this.widgetId !== this.defaultWidgetId) {
```

Comment:
I would change it to:
```suggestion
          if (!this. _isAnonymous) {
```

Reply:
no, I want to change it if you entered widgetId

Reply:
It'll be much easier to understand why